pkg/pillar/types/global.go

```diff
@@ -725,10 +746,12 @@ func NewConfigItemSpecMap() ConfigItemSpecMap {
 	configItemSpecMap.AddIntItem(Dom0DiskUsageMaxBytes, 2*1024*1024*1024,
 		100*1024*1024, 0xFFFFFFFF)
 	configItemSpecMap.AddIntItem(ForceFallbackCounter, 0, 0, 0xFFFFFFFF)
+	configItemSpecMap.AddIntItem(EveMemoryLimitInBytes, eveMemoryLimitInBytes, 0, eveMemoryLimitInBytes)
```

Comment:
Having the default be what you read from the cgroup config is fine.
But you are also setting the max to that, which means we can't manually have the controller reserve more memory for EVE. This means in particular that if the cgroup file does not exist, then we can't reserve anything. 
I think the max for the setting can be a lot larger since that would make the knob a lot more useful.

Reply:
Resolved